tekton/resources/checks/check-pr-has-kind-label.yaml

```diff
@@ -0,0 +1,203 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: triggers-minimal
```

Comment:
We might factor up roles and service accounts for triggers at some point, but until then we'll have to keep the name unique enough to avoid conflicts